src/main/java/com/codeborne/selenide/ElementsCollection.java

```diff
@@ -469,13 +469,30 @@ public ElementsCollection snapshot() {
     return new ElementsCollection(new CollectionSnapshot(collection));
   }
 
+  /**
+   * Give this collection a human-readable name
+   *
+   * Caution: you probably don't need this method.
+   * It's always a good idea to have the actual selector instead of "nice" description (which might be misleading or even lying).
```

Comment:
Maybe it's worth adding some `<p>` tags to make this JavaDoc look exactly as you meant it to be here.

Reply:
I tried to add some `<p>`, but don't really see any difference. Probably I am just doing it wrong. 

@bereg2k Don't you want to send a PR with javadoc improvements?

Reply:
Without tags. Looks nice in the "edit" form.
![image](https://user-images.githubusercontent.com/42677987/110766911-ba5f1280-8266-11eb-8e6c-acfb7442020c.png)

But not as nice in the "read" form:
![image](https://user-images.githubusercontent.com/42677987/110767042-dcf12b80-8266-11eb-8fb9-bba62dd6eb91.png)
![image](https://user-images.githubusercontent.com/42677987/110767098-ebd7de00-8266-11eb-9762-6d4a30ac5390.png)

Reply:
Even a single usage of tags makes it look as it was intended:

![image](https://user-images.githubusercontent.com/42677987/110767293-2772a800-8267-11eb-8ad7-c5f2be5a8788.png)
![image](https://user-images.githubusercontent.com/42677987/110767331-335e6a00-8267-11eb-94c7-f6b5266dd051.png)

Or even this:
![image](https://user-images.githubusercontent.com/42677987/110767445-51c46580-8267-11eb-8ae9-a9d8a7c39985.png)

![image](https://user-images.githubusercontent.com/42677987/110767476-5a1ca080-8267-11eb-864a-905f3ac1e9ec.png)

@asolntsev I'm up for the task btw, but I'm just saying, that any particular case of JavaDoc might benefit from some tagging.

Reply:
Thank you for the knowledge sharing. 
But when I wrapped every clause to `<p></p>`, it still didn't look well in the resulting JavaDoc. 

Well, I see that you are not wrapping text to `<p>` tags, but using empty `<p></p>` essentially to create line break (like `<br>`). Seems strange for me...

Anyway, it seems that quite a lot of Selenide javadoc would need this formatting improvement, so probably it's a good idea to create a dedicated pull request.